qutip/ipynbtools.py

```diff
@@ -39,10 +39,14 @@
 import qutip
 import numpy
 import scipy
-import Cython
 import matplotlib
 import IPython
 
+try:
+    import Cython
+except:
+    pass
```

Comment:
```suggestion
except ImportError:
    pass
```

For try except it is best to be specific with the type of exception to avoid unexpected bugs.